src/app/forms/custom-sections/tyres/tyres.component.ts

```diff
@@ -173,7 +173,7 @@ export class TyresComponent implements OnInit, OnDestroy, OnChanges {
   }
 
   getTyreSearchPage(axleNumber: number) {
-    const route = this.editingReason ? `../${this.editingReason}/tyre-search/${axleNumber}` : `../new-record-details/tyre-search/${axleNumber}`;
```

Comment:
Still not entirely sure what the original line was for, why it backed out a level only to go back into the same page for the create tech record bit - if someone could explain that to me in case I've broken something, that'd be great